src/app/RankStore.js

```diff
@@ -0,0 +1,12 @@
+const maxRankingSize = 3;
+
+export const getRanking = (mode) =>
+  JSON.parse(localStorage.getItem(mode)) || [];
+
+export const rankPlayer = (nickname, score, mode) => {
+  const currentRanking = getRanking(mode);
+  const newRanking = [...currentRanking, { nickname, score }]
```

Comment:
How about
```js
if (quizRanking.length > 3) {
//shorten array to 3 items
} 
```

Reply:
`export const rankPlayer = (nickname, score, mode) => {
  const currentRanking = getRanking(mode);
  let newRanking = [...currentRanking, { nickname, score }].sort(
    (a, b) => b.score - a.score,
  );
  if (newRanking.length > 3) {
    newRanking = newRanking.slice(0, maxRankingSize);
  }
  localStorage.setItem(mode, JSON.stringify(newRanking));
};`

Do you mean something like this?